docs/source/get_started/installation.rst

```diff
@@ -163,6 +199,55 @@ If you are using the Windows and the PowerShell:
     pytest -m "not slow" deepchem # optional
 
 
+From source lightweight guide
```

Comment:
Should we merge these with the existing from source guide? It seems similar in complexity I think

Reply:
Well that one uses install_deepchem_conda script and we will have to build a new conda script for an overall installation